src/pages/account/mijn-mappen/[folderSlug]/index.tsx

```diff
@@ -83,7 +83,7 @@ const AccountMyFolders: NextPage<DefaultSeoInfo> = ({ url }) => {
 	const sidebarLinks: ListNavigationFolderItem[] = useMemo(
 		() =>
 			(folders?.items || []).map((folder) => {
-				const slug = createFolderSlug(folder);
+				const slug = folder.name && createFolderSlug(folder);
 				const href = `${ROUTES.myFolders}/${slug}`;
```

Comment:
if folder.name is undefined, slug will be undefined so the route will be: mijn-folders/undefined
maybe the createFolderSlug has to be able to deal with empty folder names?